packages/venia-ui/lib/components/Postcode/postcode.css

```diff
@@ -0,0 +1,3 @@
+.root {
+    grid-area: state;
```

Comment:
We don't want connections like this in standalone components.

This would be something that the calling component would overwrite for its instance of `Postcode` using the `classes` prop.

Reply:
I updated to use `grid-area: zip` . Is that right to working?